src/screens/OtherProfile/index.js

```diff
@@ -1,6 +1,7 @@
 import * as React from 'react';
 import SimpleToast from 'react-native-simple-toast';
 import ToastMessage from 'react-native-toast-message';
+import netInfo from '@react-native-community/netinfo';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `netInfo` from `@react-native-community/netinfo` has been added but it seems like it's not being used anywhere in the code. Unused imports can lead to confusion and unnecessary increase in bundle size. Please remove it if it's not needed.
```diff
- import netInfo from '@react-native-community/netinfo';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->

Reply:
this is used